doc/For developers/Developer guide/Performance/Testing.md

```diff
@@ -40,8 +40,7 @@ Conclusions:
 
 - Enabling Hyper-V on Windows Server 2016 reduces TCP throughput by a factor of 3-4.
```

Comment:
Add information which exactly test cases support this conclusions (for this one A and B test case)